libs/application/templates/parental-leave/src/forms/ResidenceGrant.ts

```diff
@@ -20,36 +21,45 @@ export const ResidenceGrant: Form = buildForm({
   logo: Logo,
   children: [
     buildSection({
-      id: 'residentGrantApplication',
+      id: 'residenceGrant',
       title:
         parentalLeaveFormMessages.residenceGrantMessage
           .residenceGrantApplyTitle,
       children: [
         buildMultiField({
-          title: 'DateField',
-          id: 'dvalarstyrk',
+          title:
+            parentalLeaveFormMessages.residenceGrantMessage
+              .residenceGrantApplyTitle,
+          id: 'residenceGrant.multiOne',
           description:
-            'Add the date form when you wish to apply for Dvalastyrkur',
+            parentalLeaveFormMessages.residenceGrantMessage
+              .residenceGrantSelectPeriodDescription,
           space: 2,
           children: [
             buildDateField({
-              id: 'dvalarstyrk.dateFrom',
+              id: 'residenceGrant.dateFrom',
               title: 'From',
               placeholder: '',
               backgroundColor: 'blue',
               width: 'half',
             }),
             buildDateField({
-              id: 'dvalarstyrk.dateTo',
+              id: 'residenceGrant.dateTo',
               title: 'To',
               placeholder: '',
               backgroundColor: 'blue',
               width: 'half',
             }),
+            buildCustomField({
+              id: 'imagefield',
+              title: '',
+              defaultValue: 0,
+              component: 'ImageField',
+            }),
           ],
         }),
         buildFileUploadField({
-          id: 'residenceGrantApplication.fileUpload',
+          id: 'residenceGrant.fileUpload',
```

Comment:
Do we want create new object for fileUpload residenceGrant or should we have all fileUpload under fileUpload object?

Reply:
Under file upload I think. I will change that.